src/main/java/mate/academy/spring/config/DataInitializer.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.config;
+
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataInitializer {
+    @Autowired
+    private final RoleService roleService;
+    @Autowired
+    private final UserService userService;
+
+    public DataInitializer(RoleService roleService, UserService userService) {
+        this.roleService = roleService;
+        this.userService = userService;
```

Comment:
```suggestion
    private final RoleService roleService;
    private final UserService userService;

    public DataInitializer(RoleService roleService, UserService userService) {
        this.roleService = roleService;
        this.userService = userService;
```
better use autowiring by constuctor